django/test/runner.py

```diff
@@ -448,6 +450,8 @@ def __init__(self, pattern=None, top_level=None, verbosity=1,
         self.parallel = parallel
         self.tags = set(tags or [])
         self.exclude_tags = set(exclude_tags or [])
+        if not no_faulthandler:
+            faulthandler.enable(file=sys.__stderr__)
```

Comment:
`file` supports file descriptor so we can use the same workaround like `pytest`:
```python
try:
    faulthandler.enable(file=sys.stderr.fileno())
except (AttributeError, io.UnsupportedOperation):
    faulthandler.enable(file=sys.__stderr__.fileno())
```